rust/arrow/src/lib.rs

```diff
@@ -15,6 +15,12 @@
 // specific language governing permissions and limitations
 // under the License.
 
+//! A native Rust implementation of [Apache Arrow](https://arrow.apache.org), a cross-language
+//! development platform for in-memory data.
+//!
+//! Currently the project is developed and tested against nightly Rust. To learn more about the
+//! status of Arrow in Rust, see `README.md`.
```

Comment:
nit: can `README.md` be a link?

Reply:
I actually don't know how to do this in a portable way. What's the best way to link to the README?

Reply:
Link to the README on master, https://github.com/apache/arrow/blob/master/rust/arrow/README.md